recipes/billingegroup-meta/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "billingegroup-meta" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  skip: True  # [py36 or win]
```

Comment:
To skip python 3.6 you'll need to add a build requirement that is just python. e.g.:
```yaml
requirements:
  build:
    - python
```

Somewhat ironically, the only package that was produced on the CI services is:
```
Resolved dependencies, will be built in the following order: 
	billingegroup-meta-0.1.0-py36_0 (will be built: True)
                                 ^^^^
```

Adding this build requirement of python will instruct conda-build-all to produce py27 and py35 metapackages.